src/snowflake/cli/plugins/spcs/__init__.py

```diff
@@ -1,17 +1,19 @@
 import typer
 from snowflake.cli.api.commands.flags import DEFAULT_CONTEXT_SETTINGS
-from snowflake.cli.plugins.containers.compute_pool.commands import (
+from snowflake.cli.plugins.spcs.compute_pool.commands import (
     app as compute_pools_app,
 )
-from snowflake.cli.plugins.containers.jobs.commands import app as jobs_app
-from snowflake.cli.plugins.containers.services.commands import app as services_app
+from snowflake.cli.plugins.spcs.jobs.commands import app as jobs_app
+from snowflake.cli.plugins.spcs.registry.commands import app as registry_app
+from snowflake.cli.plugins.spcs.services.commands import app as services_app
 
 app = typer.Typer(
-    name="containers",
+    name="spcs",
     context_settings=DEFAULT_CONTEXT_SETTINGS,
     help="Manages Snowpark services, pools and jobs.",
```

Comment:
With moving registry here, should we update it? I'm fine with current state, just confirming

Reply:
Good point, I've updated the note to include image registries and repositories for clarity.